tasks/FileUploadTask.md

```diff
@@ -135,12 +143,13 @@ let options = {
 };
 
 // create an upload session
-const uploadTask = await MicrosoftGraph.OneDriveLargeFileUploadTask.create(client, file, options);
+const uploadTask = await MicrosoftGraph.LargeFileUploadTask.create(client, file, options);
```

Comment:
There is no `create()` function in LargeFileUpload task. It is only in OneDriveFileUpload. Please undo this for the time being. I will update this.

Reply:
The goal of this document is to describe the future state of the solution, not the current one. I'd like this spec to be as final as possible so we each have clear guidelines on what changes need to be implemented in each SDK. The Java samples also do not map to current state here, this is expected.

Reply:
LargeFileUploadTask.createUploadSession(client: Client, requestUrl, payload, headers) this is what is present.

Reply:
which is the equivalent of `graphClient.me().drive().item("blah.blob").createUploadSession()` and is present as a static method because the JS SDK doesn't have a fluent API yet, correct?

What's the method to create an upload task from the session? a ctor like `new LargeFileUploadTask(session, stream, options)` ? And then `task.upload()`?